### PR TITLE
Changelog for Jenkins 2.0

### PR DESCRIPTION
**Pull request by daniel-beck**
Does this look complete?

@reviewbybees

### CODE REVIEW

File: changelog.html [hunk 1]
```diff
@@ -55,8 +55,52 @@
...
+</div>
+
 <ul class=image>
-  <li class=>
+  <li class="rfe major">
+    New password-protected setup wizard shown on first run to guide users through installation of popular plugins and setting up an admin user.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-30749">issue 30749</a>,
+     <a href="https://issues.jenkins-ci.org/browse/JENKINS-9598">issue 9598</a>)
+  <li class="rfe major">
+    Plugin bundling overhaul: Bundled plugins are only installed if necessary when upgrading, all plugins can be uninstalled.
```
Comment (jtnord) [inline changelog.html]: implies that some plugins may be upgraded when you upgrade to 2.0 which is not the case IIUC - as there are no bundled plugins?

File: changelog.html [hunk 2]
```diff
@@ -55,8 +55,52 @@
...
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-20617">issue 20617</a>)
+  <li class="rfe major">
+    Redesigned job configuration form makes it easier to see the option hierarchy, and to navigate the form using scrolling tabs.
```
Comment (jtnord) [inline changelog.html]: NIT: they are not tabs, just look like them, but they don;t act like them.

File: changelog.html (hunk 1 above)
Comment (daniel-beck) [inline changelog.html]: @jtnord We still bundle 'detached' plugins, but they are only installed when Jenkins determines it was upgraded from a version that had the detached plugin as core feature.

File: changelog.html (hunk 2 above)
Comment (daniel-beck) [inline changelog.html]: @jtnord If you know a better name than *scrolling tabs* to describe them, I'd like to know.

Comment (jtnord) [inline changelog.html]: just leave it as redesigned job configuration form?

File: changelog.html (hunk 1 above)
Comment (jtnord) [inline changelog.html]: ok.

**Comment by jtnord** [comment]
:bee:

**Comment by jglick** [comment]
:bee:

**Pull request merged**